src/client/include/com/amazonaws/kinesis/video/client/Include.h

```diff
@@ -1071,6 +1093,14 @@ typedef struct __ClientInfo {
 
     // Time that allowed to be elapsed between the metric loggings if enabled
     UINT64 metricLoggingPeriod;
+
+    // ------------------------------ V1 compat --------------------------
+
+    // flag for automatic handling of intermittent producer
+    AUTOMATIC_STREAMING_FLAGS automaticStreamingFlags;
+
+    // period (in hundreds of nanos) at which callback will be fired to check stream
+    UINT64 callbackPeriod;

```

Comment:
Not sure if we need this to be public. There are certain implications if the customer sets different values - small or large. In one case, we will have odd consequences of sending EoFrs too frequently causing fragmentation whenever it's not desirable and in the other case this won't have any effect because the hard-timeout would kick in prior this. Recommend removing this and having a definition in an internal header

Reply:
Agreed to change named to `reservedCallbackPeriod` and use sentinel value, and during createKinesisVideoClient check if it was set by caller, if not then use INTERMITTENT_PRODUCER_PERIOD_DEFAULT.